lib/cosmos/io/json_drb_object.rb

```diff
@@ -106,7 +106,7 @@ def connect
         if !@http
           @http = HTTPClient.new
           @http.connect_timeout = @connect_timeout
-          @http.receive_timeout = @connect_timeout
+          @http.receive_timeout = nil # Allow long polling
```

Comment:
Do we ever want a receive timeout? Parameter?

Reply:
As long as we are using the HttpClient library I don't think so.  It uses ::Timeout which is bad, but which does nothing if you pass it nil.  With any actual timeout it would spin up threads and other side effects

Reply:
I am still worried about long term reliability using blocking sockets within HttpClient, but this at least fixes most issues.

Reply:
Sounds good 👍